bin/anthology/papers.py

```diff
@@ -282,6 +287,8 @@ def as_bibtex(self, concise=False):
             entries.append(("pages", self.get("pages").replace("–", "--")))
         if "xml_abstract" in self.attrib and not concise:
             entries.append(("abstract", self.get_abstract(form="latex")))
+        if self.language:
+            entries.append(("language", iso630.languages.get(part3=self.language).name))
```

Comment:
What's `iso630`? That's not defined/imported anywhere

Reply:
Whoops, fixed.